tools/appstreamcli.c

```diff
@@ -1068,7 +1068,7 @@ static int
 as_client_run_compose (const gchar *command, char **argv, int argc)
 {
 	const gchar *ascompose_exe = LIBEXECDIR "/appstreamcli-compose";
-	g_autofree const gchar **asc_argv = NULL;
```

Comment:
Why did you remove the `const`? This will most likely break the following code when `ascompose_exe` and other constant strings are assigned (we really just want to allocate the container here, not the string contents).

Reply:
Ah, the `const` will break up with compilation.